test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -435,6 +435,38 @@ private static void AssertElementsAreJoined(Element element, Element otherElemen
             Assert.AreEqual(expected, arejoined);
         }
 
+        [Test]
+        [TestModel(@".\Element\elementJoin.rvt")]
+        public void CanSuccessfullyUnjoinListOfElements()
+        {
+            var wall1 = ElementSelector.ByElementId(184176, true);
+            var wall2 = ElementSelector.ByElementId(207960, true);
+            var floor = ElementSelector.ByElementId(208259, true);
+
+            // Are joined
+            bool orignalWall1AndWall2JoinedValue = wall1.AreJoined(wall2);
+            // Are joined
+            bool orignalWall1AndFloorJoinedValue = wall1.AreJoined(floor);
+            // Are not joined
+            bool orignalWall2AndFloorJoinedValue = wall2.AreJoined(floor);
```

Comment:
also add an `Assert` here that confirms their initial values are what we expected.